### PR TITLE
Initial EKS cluster creation

### LINKED ISSUE #98: Add README on terraform codebase
### What would you like to be added?

Instructions on how to use and install the terraform to build a test infrastructure

### Why is this needed?

Help people know how to use the code base to run the test on their own infrastructure, how to access the cluster, how to ssh into the jumper etc.

### PR DESCRIPTION
**Pull request by knabben**
#### What type of PR is this?
/label kind/documentation

#### What this PR does / why we need it:
* Refactoring the cluster module to simplify the managed node groups creation on EKS and updating to Windows 2022.
* Adding README for the Terraform EKS

#### Which issue(s) this PR fixes:
Fixes #98

### CODE REVIEW

File: terraform/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+Terraform >= 1.1.0
+AWS Account with proper IAM permissions
+
+### Initializing modules
+
+Under the folder `./terraform` all the resources exists, to initizlie and download the used modules
+call, terraform with init parameter:
+
+```shell
+$ terraform init
```
Comment (ameukam) [inline terraform/README.md]:
```suggestion
$ terraform init -backend false
```

The backend will be ignored

**Comment by ameukam** [comment]
Why are we dropping the config for the jumper ?

**Comment by knabben** [comment]
> Why are we dropping the config for the jumper ?

I think I missed the jumper setup, is `data "aws_ami" "ubuntu_2204"` bringing the EC2 or just setting the data?

**Comment by knabben** [comment]
@marciogmorales will take over from here.

**Pull request merged**